docs/checks/unusualLayerTagsCheck.md

```diff
@@ -0,0 +1,24 @@
+# UnusualLayerTagsCheck
+
+#### Description
+
+The purpose of this check is to identify layer tag values when accompanied by invalid tunnel and bridge tags.
+
+#### Live Examples
+
+1. Line [id:521569767](https://www.openstreetmap.org/way/521569767) has _Layer_ Tag > 0, and is within tunnel=BUILDING_PASSAGE.
+2. Line [id:479328850](https://www.openstreetmap.org/way/479328850) has _Layer_ Tag < 0, and crosses bridge=YES.
+
+#### Code Review
+
+This check is intended to validate layer tag values when accompanied by tunnel and bridge tags which meet any of the following four requirements.
+
+    1. Layer tag value should be an integer, ranging from -5 to 5, excluding 0 [LayerTag.java](https://github.com/osmlab/atlas/blob/dev/src/main/java/org/openstreetmap/atlas/tags/LayerTag.java)
```

Comment:
The links for these bullet points arent showing because this is rendering as a code block